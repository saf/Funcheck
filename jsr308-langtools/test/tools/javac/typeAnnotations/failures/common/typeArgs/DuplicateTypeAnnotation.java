/*
 * @test /nodynamiccopyright/
 * @bug 6843077
 * @summary check for duplicate annotations
 * @author Mahmood Ali
 * @compile/fail/ref=DuplicateTypeAnnotation.out -XDrawDiagnostics -source 1.7 DuplicateTypeAnnotation.java
 */

class DuplicateTypeAnno<K> {
  DuplicateTypeAnno<@A @A String> l;
}

@interface A { }
