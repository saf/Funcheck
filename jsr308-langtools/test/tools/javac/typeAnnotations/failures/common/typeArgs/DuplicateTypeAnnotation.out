DuplicateTypeAnnotation.java:10:24: compiler.err.duplicate.annotation
1 error
